index.html

```diff
@@ -3,136 +3,328 @@
 
 <head>
   <meta charset="utf-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-  
-    <!-- Bootstrap CSS -->
-    <link rel="stylesheet"
-        href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"
-        integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T"
-        crossorigin="anonymous">
-    <link rel="stylesheet"
-        href="main.css">
-    <title>QuestBoard
-    </title>
+  <meta name="viewport"
+    content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+  <!-- Bootstrap CSS -->
+  <link rel="stylesheet"
+    href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"
+    integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T"
+    crossorigin="anonymous">
+  <link rel="stylesheet"
+    href="main.css">
+  <title>QuestBoard
+  </title>
 </head>
 
 <body>
   <header>
-    <div id="carouselIndicators" class="carousel slide" data-ride="carousel">
-      <ol class="carousel-indicators">
-        <li data-target="#carouselIndicators" data-slide-to="0" class="active"></li>
-        <li data-target="#carouselIndicators" data-slide-to="1"></li>
-        <li data-target="#carouselIndicators" data-slide-to="2"></li>
-        <li data-target="#carouselIndicators" data-slide-to="3"></li>
+    <div
+      id="carouselIndicators"
+      class="carousel slide"
+      data-ride="carousel">
+      <ol
+        class="carousel-indicators">
+        <li
+          data-target="#carouselIndicators"
+          data-slide-to="0"
+          class="active"></li>
+        <li
+          data-target="#carouselIndicators"
+          data-slide-to="1">
+        </li>
+        <li
+          data-target="#carouselIndicators"
+          data-slide-to="2">
+        </li>
+        <li
+          data-target="#carouselIndicators"
+          data-slide-to="3">
+        </li>
       </ol>
-      <div class="carousel-inner">
-        <div class="carousel-item active">
-          <img src="dai.jpg" class="d-block vw-100 vh-100 scale" alt="...">
-          <div class="carousel-caption d-none d-md-block">
-            <h1 class="carousel-heading">Meet fun people</h1>
-            <p class="carousel-paragraph">optional rating of other gamers</p>
+      <div
+        class="carousel-inner">
+        <div
+          class="carousel-item active">
+          <img src="dai.jpg"
+            class="d-block vw-100 vh-100 scale"
+            alt="...">
+          <div
+            class="carousel-caption d-none d-md-block">
+            <h1
+              class="carousel-heading">
+              Meet fun people
+            </h1>
+            <p
+              class="carousel-paragraph">
+              optional rating
```

Comment:
It looks like a bad practice - Robert reformatted my section code to prevent the abuse of enters.

<div
        class="carousel-inner">
        <div
          class="carousel-item active">

should look like this:

<div class="carousel-inner">
    <div class="carousel-item active">